interfaces/micropython/SerialWombat_mp_i2c.py

```diff
@@ -9,7 +9,7 @@ class SerialWombatChip_mp_i2c(SerialWombat.SerialWombatChip):
     def __init__(self,i2c_port,address):
         self.i2c = i2c_port
         self.address = address
-        SerialWombat.SerialWombatChip.__init__(self)
+        super().__init__()
```

Comment:
In that case, the SerialWombatChip's init is setting self.address to 0, which overwrites the value set here on line 11.
If you change this to:
```
    def __init__(self,i2c_port,address):
        super().__init__()
        self.i2c = i2c_port
        self.address = address
```
you won't have that problem

Reply:
I noticed that as well.  I fixed it after I merged your pr.   Thanks again!

Reply:
Great! I'm glad I could help, I appreciate all the work you've put into this so far.